py/server/deephaven/table.py

```diff
@@ -362,9 +366,24 @@ def _j_py_script_session() -> _JPythonScriptSession:
 _numpy_type_codes = ["i", "l", "h", "f", "d", "b", "?", "U", "O"]
 
 
+def _array_component_type(fn) -> str:
+    sig = inspect.signature(fn)
+    t = sig.return_annotation
+    if isinstance(t, _GenericAlias) and issubclass(t.__origin__, Sequence):
+        return np.dtype(t.__args__[0]).char
+
+    if sys.version_info.minor > 8:
```

Comment:
what happens if the python version is `<=3.8`?  We just don't do numpy?  If so, add a comment.